src/glob/mod.rs

```diff
@@ -1448,11 +1449,6 @@ mod test {
         "invalid domain character"
       );
     }
-    // error for invalid file url
-    if cfg!(windows) {
-      let err = PathOrPattern::from_relative(&cwd, "file:///raw.githubusercontent.com%2Fdyedgreen%2Fdeno-sqlite%2Frework_api%2Fmod.ts").unwrap_err();
-      assert_eq!(format!("{:#}", err), "Could not convert specifier to file path.\n  Specifier: file:///raw.githubusercontent.com%2Fdyedgreen%2Fdeno-sqlite%2Frework_api%2Fmod.ts");
-    }
```

Comment:
This passes now on windows, which doesn't seem like a big deal so I deleted this test.